packages/embed/README.md

```diff
@@ -117,58 +117,59 @@ Closing and opening a typeform in modal window will restart the progress from th
+| [opacity](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/widget-inline)                                  | number                      | form background opacity, number from 0 (fully transparent) 100 (fully opaque)                                                                                                                                                                                                                          | `100`                                                         |
+| [autoFocus](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/widget-focus)                                 | boolean                     | enable form auto focus when loaded                                                                                                                                                                                                                                                                     | `false`                                                       |
+| [open](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/load-js)                                           | string                      | open embed based on user action (see below)                                                                                                                                                                                                                                                            | `undefined`                                                   |
+| [openValue](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/time-js)                                      | number                      | based on `open` (see below)                                                                                                                                                                                                                                                                            | `undefined`                                                   |
+| [preventReOpenOnClose](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/prevent-reopen-on-close-html.html) | boolean                     |                                                                                                                                                                                                                                                                                                        | `false`                                                       |
```

Comment:
not sure if the link is correct here. following the other example this should be good. did not see how the code gets copied into codesandbox

Reply:
Yes. The link will exist only after your PR is merged. Eg. [the preselect link](https://codesandbox.io/p/sandbox/github/Typeform/embed-demo/tree/main/demo-html/preselect-html?file=%2Findex.html) did not exist this morning either, but after the PR was merged the demo was built in https://github.com/Typeform/embed-demo and now it works.